### DISCUSSION TITLE
How do you manually get and set the rowCount?

### DISCUSSION THREAD

**Discussion by cedoromal** [TanStack/table#5492] [Q&A]
I tried what was written in the [docs](https://tanstack.com/table/latest/docs/api/features/pagination#rowcount) to set the rowCount but it doesn't work. 

![image](https://github.com/TanStack/table/assets/30488557/e96ae3aa-0d0c-4c92-9376-87f7d99ecf3a)

It says that 'rowCount' does not exist in type 'TableOptions<Partial<...>>'

and while we're at it, how do you retrieve the rowCount from the table?

EDIT: Btw I'm using @tanstack/react-table@8.10.7

**Comment by fredmorais** [comment]
Did you end up figuring it out? I'm having the same problem
